static-asset-loader.js

```diff
@@ -14,8 +14,7 @@ module.exports = function(content) {
     const prefix = query.prefix || "";
 
     const hash = loaderUtils.interpolateName(this.context, `[hash]`, {
-        context: this.options.context,
-        regExp: this.options.regExp,
+        context: this.rootContext || this.options.context,
```

Comment:
is `|| this.options.context` needed at all? I thought `this.rootContext` replaces it in webpack 4.

Reply:
I just kept the old one to make it backwards compatible.